streamz/tests/test_sinks.py

```diff
@@ -62,7 +62,7 @@ def test_sink_to_textfile_closes():
 
 def test_sink_destroy():
     source = Stream()
-    sink = source.sink(lambda x: None)
+    sink = Sink(source)
     ref = weakref.ref(sink)
     sink.destroy()
     del sink
```

Comment:
`assert sink not in sinks._global_sinks`

Reply:
OK, will add. Didn't add this because I figured we only cared about the end result: that the sink is garbage collected.